<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="assets/favicon.ico">

    <title>Bioinformatic Dissection of High-Level Cellular Behavior - Molecular &amp; Cellular Biology - News Archives - Harvard University</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/bootstrap-4.1.3/dist/css/bootstrap.min.css" rel="stylesheet">

    <link href="assets/css/mcb_archive.css" rel="stylesheet">
    <style>
        .container {
            font-size:95%;
        }
    </style>
  </head>

  <body>
<div class="container">
    <div class="row" style="background-color:#c5cdd7; padding:10px 0;">
          <div class="col-12 text-center">
            <a class="blog-header-logo text-dark"  href="index.html">
                <img src="assets/images/mcb_header2.png" class="img-fluid" alt="Molecular and Cellular Biology" /></a>
          </div>
    </div>
</div>

<div class="container" id="news-page-body">

    <div class="row"  id="letter_or_year_menu">
    <div class="col-5 text-left">
        
            <a href="story-2005-3366-faithful-chromosome-segregation.html">&lt;&lt;</a>
        
        &nbsp;
        
            <a href="story-2005-3366-faithful-chromosome-segregation.html">Faithful Chromosome Segregation</a>
        

    </div>
    <div class="col-2 text-center">
        &nbsp; | &nbsp;
    </div>
    <div class="col-5 text-right">
        
            <a href="story-2005-3364-lichtman-at-work-eyes-wide-open.html">Lichtman at Work: Eyes Wide Open</a>
        
        &nbsp;
        
            <a href="story-2005-3364-lichtman-at-work-eyes-wide-open.html">&gt;&gt;</a>
        
    </div>
</div>


    <div class="row">
        <div class="col-12 text-left" >
            <br />
            <p>
                <a href="news_year_2017.html">News Archives</a> :
                <a href="news_year_2005.html">2005</a> :
                Bioinformatic Dissection of High-Level Cellular Behavior
        </div>
    </div>


    <!-- title row -->
    <div class="row">
        <div class="col-12 text-left" >
            <h2 class="title">Bioinformatic Dissection of High-Level Cellular Behavior</h2>

            
            <p>by 
    
    Patricia Thomas
                
            </p>
            
            <p>January 19, 2005</p>
        </div>

        <!-- start: story content --><div class="col-12 text-left">
            <div class="news_img_left"><img src="news_images/editor_uploads/2005/01/deBivort1.jpg" width="200" height="232"><br>
                      <b class="smaller">Ben de Bivort</b>
                  <b class="smaller">Figures<br></b>
                      <p><a href="deBivortFig1.html"><img src="news_images/editor_uploads/2005/01/deBivort2small.gif" width="186" height="176" border="1"></a></p>
                      <p><a href="deBivortFig2.html"><img src="news_images/editor_uploads/2005/01/deBivort3small.jpg" width="186" height="195" border="1"></a></p></div>
		    <p>Although microarray technology is wildly popular among biologists, even its biggest fans are sometimes overwhelmed by the sense of having too much information. This is especially frustrating for systems biologists trying to spot patterns in data sets that can seem as vast as the Milky Way. MCB graduate student Benjamin de Bivort became interested in this problem in early 2003, during a short course on complex systems taught by MCB associate Yaneer Bar-Yam, president of the <a href="http://necsi.org/">New England Complex Systems Institute</a>, an independent educational and research institution located in Cambridge, MA.</p>
		    <p>If de Bivort could sort microarray data into a number of &quot;bins&quot; that was small enough to be mathematically tractable, yet big enough to yield robust biological results, he thought it would be possible to observe interactions among a cell&rsquo;s functional components. In the December 21, 2004, &nbsp;issue of <i>Proceedings of the National Academy of Sciences, </i>he and two colleagues report using 12 gene groups to identify the functional and regulatory dynamics of a cell&rsquo;s overall response to assorted small molecules, cytokines, and signaling proteins. (See <a href="http://www.pnas.org/cgi/content/abstract/101/51/17687">de Bivort et al., &quot;Dynamics of cellular level function and regulation derived from murine expression array data.&quot;</a>)</p>
		    <p>In addition, these findings could be used to predict how major functional units within the cell will respond to new mutations, experimental drugs, or other perturbations, de Bivort says. His coinvestigators in this work were Bar-Yam and Sui Huang, a postdoctoral fellow in vascular biology at Children&rsquo;s Hospital Boston and Harvard Medical School.</p>
		    <p>Raw microarray data, which came from the Alliance for Cellular Signaling (AfCS)&mdash;a multi-institutional consortium designed to answer global questions about cell-signalling networks&mdash;exposed approximately 16,000 cDNAs from murine B lymphocytes to 33 different conditions, and measured expression levels at four different times (0.5 hour, 1 hour, 2 hours, 4 hours). De Bivort used the resulting expression profiles, consisting of 132 values for each cDNA, to put the genes into 12 functional groups. Gene ontology notations enabled the researchers to identify categories of functional processes for each of these 12 &quot;megamodules,&quot; such as ATP synthesis or chromatin unpacking, and time-step data made it possible to chart how influences exerted by each group changed over time. </p>
		    <p>&quot;By using large groups and averaging data, we were able to see what&rsquo;s really going on,&quot; says de Bivort. He compares this strategy to letting your eyes drift out of focus so you can see a hidden image of Mickey Mouse emerge from what looks like a uniform geometric pattern.</p>
		    <p>For the most part, the megamodules turned one another on and off in predictable ways. For example, a group rich in aerobic respiration genes&mdash;essential for generating ATP needed for transcription&mdash;exerted a strong activating influence on other gene groups. In contrast, megamodules packed with genes for ATP-consuming behaviors, such as DNA replication and chromatin remodeling, inhibited transcription.&nbsp; </p>
		    <p>Two unexpected insights, however, did surface. One sheds light on how global transcription is regulated to maintain homeostasis, a phenomenon easy to see in a single endocrine circuit with a negative feedback loop but poorly understood at the systems level. De Bivort&rsquo;s analysis showed that pairs of megamodules that are co-regulated respond in opposite ways to the same experimental condition, balancing one another and keeping the cell on an even keel.</p>
		    <p>The second surprise involves short-term activation/long-term repression, which de Bivort describes as &quot;a fairly simple idea&mdash;a component in a lattice or time series will encourage nearby components that are similar but discourage or repress dissimilar ones that are further away. It&rsquo;s like kids on a playground: the ones with Pokemon cards will stick together and will push away kids with Beanie Babies.&quot; The analysis showed that the only megamodule over-endowed with cell-cycle genes oscillated over time, rather than space, first activating other groups and then later inhibiting them. </p>
		    <p>What is most likely to interest biomedical researchers, of course, is the predictive value of this analytic method. Using expression profiles from 27 of the experimental exposures, de Bivort and his team were able to correctly predict how the cell would respond to the other 6 ligands. &quot;This means we could predict the response of the cell to any perturbation, not just the 33 in the data set,&quot; he says. The interactions among the 12 gene groups are &quot;universal,&quot; de Bivort says, and they should hold true for other cell types and ligands. </p>
		    Although some will no doubt use this model to test experimental drugs, de Bivort won&rsquo;t be among them. He is now a third-year graduate student in the laboratory of <a href="/Faculty/faculty_profile.php?f=sam-kunes">Professor Sam Kunes</a>, pursuing thesis research that won the Ernie Peralta Award last October. Turning away from bioinformatics for the time being, de Bivort is using traditional developmental and behavioral genetics methods to study visual learning in drosophila.
<p style="clear:left"></p>
        </div><!-- end: story content -->
    </div>
    <!-- end: title row-->


</div><!-- end: main container -->
<div class="container" id="footer-container">
    <div class="row" id="news-footer">
        <div class="col-12 text-left">
                Copyright &copy; The President and Fellows of Harvard College. Created September 2018.
        </div>
    </div>
</div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->

    <!--script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script-->
    <script src="assets/js/vendor/jquery-slim.min.js"></script>
    <script src="assets/js/vendor/popper.min.js"></script>
    <script src="assets/bootstrap-4.1.3/dist/js/bootstrap.min.js"></script>
    <script src="assets/js/vendor/holder.min.js"></script>
    

  </body>
</html>
