<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="assets/favicon.ico">

    <title>How Not to Wear Out a Welcome - Molecular &amp; Cellular Biology - News Archives - Harvard University</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/bootstrap-4.1.3/dist/css/bootstrap.min.css" rel="stylesheet">

    <link href="assets/css/mcb_archive.css" rel="stylesheet">
    <style>
        .container {
            font-size:95%;
        }
    </style>
  </head>

  <body>
<div class="container">
    <div class="row" style="background-color:#c5cdd7; padding:10px 0;">
          <div class="col-12 text-center">
            <a class="blog-header-logo text-dark"  href="index.html">
                <img src="assets/images/mcb_header2.png" class="img-fluid" alt="Molecular and Cellular Biology" /></a>
          </div>
    </div>
</div>

<div class="container" id="news-page-body">

    <div class="row"  id="letter_or_year_menu">
    <div class="col-5 text-left">
        
            <a href="story-2008-3454-wnts-on-the-brain-requirement-of-wnt-signaling-in-cns-vasculature-development.html">&lt;&lt;</a>
        
        &nbsp;
        
            <a href="story-2008-3454-wnts-on-the-brain-requirement-of-wnt-signaling-in-cns-vasculature-development.html">Wnts on the Brain:  Requirement of Wnt Signaling in CNS Vasculature Development </a>
        

    </div>
    <div class="col-2 text-center">
        
    </div>
    <div class="col-5 text-right">
        
        &nbsp;
        
    </div>
</div>


    <div class="row">
        <div class="col-12 text-left" >
            <br />
            <p>
                <a href="news_year_2017.html">News Archives</a> :
                <a href="news_year_2008.html">2008</a> :
                How Not to Wear Out a Welcome
        </div>
    </div>


    <!-- title row -->
    <div class="row">
        <div class="col-12 text-left" >
            <h2 class="title">How Not to Wear Out a Welcome</h2>

            
            <p>by 
    
    Matthew Michael
                
            </p>
            
            <p>December 12, 2008</p>
        </div>

        <!-- start: story content --><div class="col-12 text-left">
            <div class="news_img_right"  style="width:410px;"><img src="news_images/editor_uploads/2008/12/michael3_12-12-08.jpg" width="410" height="292"><br>
                  <b class="smaller">Seung-Hwan Kim and Matthew Michael</b></div>
		  <p>Just as dinner party guests who linger at the table long after coffee and dessert can strain the patience of their hosts, an over-extended stay at the replication fork by certain DNA polymerases can have deleterious consequences for their host, the cell.  During DNA replication, the vast majority of the chromosome is replicated by the so-called "replicative" polymerases, pols delta and epsilon.  These enzymes display both high processivity and high fidelity (one mistake per 100,000 nucleotides synthesized), two characteristics desirable in a polymerase whose job is to duplicate the genome in a timely manner.  High-fidelity replication is ensured by the proofreading activity inherent in pols delta and epsilon, as well as through a spatially-constrained active site that disallows mismatched pairing in the catalytic core of the enzyme.  While pols delta and epsilon are quite proficient at replicating DNA, they run into trouble when encountering a chemically altered, or damaged, base on the template strand.  Such encounters cause polymerase stalling, and unless the problem is resolved, progression through S phase comes to a grinding halt.  Fortunately, cells have at their disposal a specialized class of DNA polymerases, so-called translesion polymerases, which can rescue a stalled replicative polymerase.  Owing to a more spacious active site, translesion polymerases accept damaged template nucleotides into their catalytic core, and can thereby copy DNA across a template lesion.  Translesion synthesis allows replication to continue, and affords the cell the luxury of postponing its repair of the damage until after the DNA has been replicated.  As with most luxuries, however, a fee is extracted, and in the case of translesion synthesis the price is the possibility of mutations.  Unlike replicative polymerases, translesion polymerases have low fidelity (1 mistake per 250 nucleotides copied), and this is due to both a lack of proofreading activity as well as the more open active site, which can allow mismatched pairing during replication.  The challenge for the cell, therefore, is to invite the translesion polymerase to the replication fork so that it may copy DNA across a lesion, but to make sure that the stay is not overextended, and that the translesion polymerase does not continue to replicate undamaged DNA after it has copied the damaged base.	</p>
		  <p>Recent work from our laboratory has provided an important clue as to how occupancy of a translesion polymerase at the replication fork is limited to replication across a lesion.  Working with early <em>C. elegans</em> embryos, we found that when the translesion polymerase pol eta replaces a replicative polymerase at a site of damage, it is immediately SUMOylated.  The attachment of SUMO to pol eta allows it to perform translesion synthesis.  When the task is complete, the SUMO moieties are hydrolyzed, and this allows pol eta to become susceptible to ubiquitin-mediated proteolysis.  Our studies show that, for every occurrence of translesion synthesis, pol eta experiences a cycle of SUMOylation, de-SUMOylation, and then ubiquitin-mediated destruction.  In support of this model, we observed that DNA damage triggers both SUMOylation and the gradual destruction of wild-type pol eta.  By constrast, a mutant pol eta that cannot associate with replication forks is neither SUMOylated nor degraded during a DNA damage response.  This shows that pol eta SUMOylation and destruction occur at the replication fork.  How are SUMOylation and destruction related to one another?  The answer to this question came when we inactivated the SUMO ligase that targets pol eta.  Under these conditions, pol eta is very rapidly degraded, even when low amounts of DNA damage are present.  This shows that the role of SUMOylation is to provide a transient level of protection to pol eta, after it joins the replication fork.  This protection is not permanent, as we observed damage-induced destruction of pol eta in wild-type cells; however, in the wild-type pol eta, destruction is slower than that which occurs in cells lacking the SUMO ligase.  These and other observations reported in our paper show that the fate of pol eta upon recruitment to the replication fork is destruction, and that the SUMO ligase determines when, whether before or after translesion synthesis, the destruction occurs.  The pathway that we have discovered thus explains how cells avoid an over-extended stay of pol eta at the replication fork - when the time has come for it to go, it is destroyed. While this may be a questionable strategy for dealing with dawdling dinner party guests, in the case of the cell, it is an efficient means of avoiding mutations that could be produced if the low-fidelity translesion polymerases were allowed to replicate DNA past the damaged base.		    </p>
		  <p>&nbsp;</p>
		  <p>Read more in <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(08)00812-5"><em>Molecular Cell</em></a></p>

        </div><!-- end: story content -->
    </div>
    <!-- end: title row-->


</div><!-- end: main container -->
<div class="container" id="footer-container">
    <div class="row" id="news-footer">
        <div class="col-12 text-left">
                Copyright &copy; The President and Fellows of Harvard College. Created September 2018.
        </div>
    </div>
</div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->

    <!--script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script-->
    <script src="assets/js/vendor/jquery-slim.min.js"></script>
    <script src="assets/js/vendor/popper.min.js"></script>
    <script src="assets/bootstrap-4.1.3/dist/js/bootstrap.min.js"></script>
    <script src="assets/js/vendor/holder.min.js"></script>
    

  </body>
</html>
