<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="assets/favicon.ico">

    <title>Chromatin &mdash; A One-Stop Engineering Solution for Compacting DNA and Parameterizing Gene Behavior - Molecular &amp; Cellular Biology - News Archives - Harvard University</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/bootstrap-4.1.3/dist/css/bootstrap.min.css" rel="stylesheet">

    <link href="assets/css/mcb_archive.css" rel="stylesheet">
    <style>
        .container {
            font-size:95%;
        }
    </style>
  </head>

  <body>
<div class="container">
    <div class="row" style="background-color:#c5cdd7; padding:10px 0;">
          <div class="col-12 text-center">
            <a class="blog-header-logo text-dark"  href="index.html">
                <img src="assets/images/mcb_header2.png" class="img-fluid" alt="Molecular and Cellular Biology" /></a>
          </div>
    </div>
</div>

<div class="container" id="news-page-body">

    <div class="row"  id="letter_or_year_menu">
    <div class="col-5 text-left">
        
            <a href="story-2008-3481-ruth-lehman-presents-2008-bloch-lecture.html">&lt;&lt;</a>
        
        &nbsp;
        
            <a href="story-2008-3481-ruth-lehman-presents-2008-bloch-lecture.html">Ruth Lehman Presents  2008 Bloch Lecture</a>
        

    </div>
    <div class="col-2 text-center">
        &nbsp; | &nbsp;
    </div>
    <div class="col-5 text-right">
        
            <a href="story-2008-3479-huda-zoghbi-to-give-prather-lectures-may-789.html">Huda Zoghbi to give Prather Lectures May 7,8,9</a>
        
        &nbsp;
        
            <a href="story-2008-3479-huda-zoghbi-to-give-prather-lectures-may-789.html">&gt;&gt;</a>
        
    </div>
</div>


    <div class="row">
        <div class="col-12 text-left" >
            <br />
            <p>
                <a href="news_year_2017.html">News Archives</a> :
                <a href="news_year_2008.html">2008</a> :
                Chromatin &mdash; A One-Stop Engineering Solution for Compacting DNA and Parameterizing Gene Behavior
        </div>
    </div>


    <!-- title row -->
    <div class="row">
        <div class="col-12 text-left" >
            <h2 class="title">Chromatin &mdash; A One-Stop Engineering Solution for Compacting DNA and Parameterizing Gene Behavior</h2>

            
            <p>by 
    
    Felix Lam
                
            </p>
            
            <p>April 24, 2008</p>
        </div>

        <!-- start: story content --><div class="col-12 text-left">
            	<div class="news_img_right" style="width:410px;"><img src="news_images/editor_uploads/2008/04/oshea-5.jpg" width="410" height="292"><br>
                  <b class="smaller"><a href="http://www.mcb.harvard.edu/Faculty/faculty_profile.php?f=erin-oshea">Erin O'Shea</a>, and Felix Lam</b></div>
		  <p>
 If stretched end-to-end, the DNA inside a single human cell would measure approximately 2-3 meters.  A fiber of this length is physically compacted a million-fold by a hierarchy of packing proteins to fit into the confines of a cell's nucleus.  At the gene level, 150 base pair segments of DNA are compacted by being spooled around protein particles known as nucleosomes.  However, not unlike computer-based compression methods, the gain in space efficiency comes at a cost &#151; the information contained within is less accessible.  Indeed, at any given moment, large swaths of the genome lie inert in a tightly bundled form known as heterochromatin.  Yet even transcriptionally-available genes do not have much of their DNA completely accessible.  Studies probing the regulatory promoter region of genes in budding yeast found that small 100-150 base pair stretches of DNA are always maintained in an un-spooled state.  To activate a particular gene, transcription factors typically bind recognition sequences in the un-spooled segment.  This then triggers the surrounding DNA to un-spool and to become accessible to additional transcription components.  These permanently exposed binding sequences thus provide a means for the cell to locally &ldquo;decompress&rdquo; DNA on-demand to selectively access genetic information.</p>
		  <p> What our lab has uncovered, however, is that this local decompaction is far more sophisticated than was previously thought.  Continuing the work in budding yeast, it turns out that binding sequences in cells are strategically blocked or exposed in order to achieve quantitatively precise gene responses.  For example, by varying the binding affinity of recognition sequences in the un-spooled segment, cells have the ability to fine-tune when a gene will activate.  In fact, by analyzing the promoter architecture of a collection of genes in the phosphate starvation pathway, we were able to accurately predict the existence of two broad gene subclasses, each of which activate at a distinct level of external stress.</p>
		  <p> While this clarifies the role of exposed sequences, what about the numerous binding sequences that normally remain inaccessible within spooled segments?  What we've found is that these hidden sites serve primarily to scale up the total gene expression strength.  Interestingly, because these sites collectively come into play only after the gene has first been triggered through the exposed sequences, a gene's threshold can be programmed largely independently from its expression strength.		    </p>
		  <p>The fact that these two gene performance attributes are decoupled from one another has a number of fundamental implications.  From an evolutionary standpoint, it suggests that cells are able to modify a gene's point of activation or its expression capacity simply by mutating DNA binding sequences in either un-spooled or spooled segments.  From a design standpoint, it suggests that nature has solved a principal engineering problem of how to modularize gene properties.  Were it not for nucleosomes, changing a gene's threshold would simultaneously affect its expression strength.  However, with nucleosomes and the ability to strategically expose particular binding sequences, cells have the power to individually tune each property, giving them more flexibility to adapt to a greater spectrum of selective pressures at the DNA level.          </p>
		  <p>April 16, 2008: Advance Online Publication in <a href="http://www.nature.com.ezp1.harvard.edu/nature/journal/vaop/ncurrent/full/nature06867.html"><em>Nature</em></a></p>

        </div><!-- end: story content -->
    </div>
    <!-- end: title row-->


</div><!-- end: main container -->
<div class="container" id="footer-container">
    <div class="row" id="news-footer">
        <div class="col-12 text-left">
                Copyright &copy; The President and Fellows of Harvard College. Created September 2018.
        </div>
    </div>
</div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->

    <!--script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script-->
    <script src="assets/js/vendor/jquery-slim.min.js"></script>
    <script src="assets/js/vendor/popper.min.js"></script>
    <script src="assets/bootstrap-4.1.3/dist/js/bootstrap.min.js"></script>
    <script src="assets/js/vendor/holder.min.js"></script>
    

  </body>
</html>
