<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="assets/favicon.ico">

    <title>A Neuronal Bar Code in 3D [Gaudet Lab] - Molecular &amp; Cellular Biology - News Archives - Harvard University</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/bootstrap-4.1.3/dist/css/bootstrap.min.css" rel="stylesheet">

    <link href="assets/css/mcb_archive.css" rel="stylesheet">
    <style>
        .container {
            font-size:95%;
        }
    </style>
  </head>

  <body>
<div class="container">
    <div class="row" style="background-color:#c5cdd7; padding:10px 0;">
          <div class="col-12 text-center">
            <a class="blog-header-logo text-dark"  href="index.html">
                <img src="assets/images/mcb_header2.png" class="img-fluid" alt="Molecular and Cellular Biology" /></a>
          </div>
    </div>
</div>

<div class="container" id="news-page-body">

    <div class="row"  id="letter_or_year_menu">
    <div class="col-5 text-left">
        
            <a href="story-2015-3838-summer-school-program-research-in-paris-lue-and-viel.html">&lt;&lt;</a>
        
        &nbsp;
        
            <a href="story-2015-3838-summer-school-program-research-in-paris-lue-and-viel.html">Summer School Program: Research in Paris [Lue and Viel]</a>
        

    </div>
    <div class="col-2 text-center">
        &nbsp; | &nbsp;
    </div>
    <div class="col-5 text-right">
        
            <a href="story-2015-3840-establishing-cell-fate-with-a-pole-vaulting-phosphatase-losick-lab.html">Establishing Cell Fate with a Pole-Vaulting Phosphatase [Losick Lab]</a>
        
        &nbsp;
        
            <a href="story-2015-3840-establishing-cell-fate-with-a-pole-vaulting-phosphatase-losick-lab.html">&gt;&gt;</a>
        
    </div>
</div>


    <div class="row">
        <div class="col-12 text-left" >
            <br />
            <p>
                <a href="news_year_2017.html">News Archives</a> :
                <a href="news_year_2015.html">2015</a> :
                A Neuronal Bar Code in 3D [Gaudet Lab]
        </div>
    </div>


    <!-- title row -->
    <div class="row">
        <div class="col-12 text-left" >
            <h2 class="title">A Neuronal Bar Code in 3D [Gaudet Lab]</h2>

            
            <p>by 
    
    Rachelle Gaudet&nbsp;and 
    John Nicoludis
                
            </p>
            
            <p>October 16, 2015</p>
        </div>

        <!-- start: story content --><div class="col-12 text-left">
            <div class="FloatL" style="width:360px;">
	<img alt="" src="news_images/editor_uploads/2015/10/2015_gaudet_structure_4165.png" style="width: 360px; height: 330px;" />John M. Nicoludis (l) and <a href="https://www.mcb.harvard.edu/mcb/faculty/profile/rachelle-gaudet/">Rachelle Gaudet</a></div>
<p>
	As the brain develops, neurons play a complicated game of Twister: they have to keep their neuronal processes from getting tangled up and improperly wired. Part of the solution to this tangling problem is that the branching processes of neurons exhibit self-avoidance: dendrites of the same neuron tend to not contact each other. In higher animals, the clustered protocadherin proteins mediate this self-avoidance. Three genes clusters encode 53 clustered protocadherin isoforms in humans. Each isoform has specificity for homophilic interactions, preferring to interact with another copy of itself versus all other isoforms. This means that each isoform can act as identity signal for a cell. But 53 isoforms alone are not enough to generate the diversity of identities required for the complex connections of the brain. Several isoforms are expressed stochastically in each cell and work together to produce a unique identity marker that is the combination of expressed isoforms. This combinatorial behavior provides the potential for billions of unique barcodes for every neuron. Therefore, the self-avoidance decision uses an &lsquo;AND&rsquo; logic gate, where homophilic interaction of each individual isoform must be satisfied for self-avoidance to be signaled.<br />
	<br />
	Interestingly, the <a href="http://www.nature.com/nature/journal/v524/n7564/full/nature14668.html">recently reported octopus genome</a> contains a total of 168 protocadherin isoforms, more than 10 times as many as other invertebrates and more than twice as many as in mammals. These protocadherins are highly expressed in their complex brain, the most complex of all mollusks. This suggests that an expansion of the number of protocadherin genes to provide mechanisms for neural network organization has occurred more than once in evolution.<br />
	<br />
	In a paper just published in <a href="http://www.sciencedirect.com/science/article/pii/S0969212615003755">Structure</a> in collaboration with Debora Mark&rsquo;s lab (Harvard Medical School), we reveal the molecular details of the homophilic interactions between clustered protocadherin that mediates self-avoidance in neurons. We obtained 3D atomic-resolution structures, which we then corroborated by sequence analyses that expose the evolutionary signatures of specificity that is needed for cells to discriminate self from non-self.<br />
	<br />
	Clustered protocadherins are membrane-bound proteins at the cell surface with six extracellular cadherin repeat domains, numbered 1 through 6 from N- to C-terminus, or furthest away from the cell to closest. Homophilic specificity is determined through interactions in repeats 2 and 3. We determined the 3D atomic structure of protein fragments containing repeats 1-3 for two different isoforms. In both structures, we observed dimers with two protocadherin molecules lying antiparallel to each other, with interactions between repeats 2 and 3. This extended antiparallel interface is reminiscent of the &ldquo;handshake&rdquo; interaction between two unusual cadherins that form &ldquo;tip link&rdquo; structures in the inner ear that are essential to detect sound waves, an <a href="http://www.nature.com/nature/journal/v492/n7427/full/nature11590.html">interaction revealed in atomic detail</a> by the Gaudet and Corey labs in 2012. Importantly, the clustered protocadherin interaction is distinct from the tip-to-tip interactions between first repeats seen in the classical cadherins that form adhesions between cells.<br />
	<br />
	We then used bioinformatics to identify amino acid residues within clustered protocadherin protein sequences that are responsible for homophilic specificity. Such residues should be highly divergent in sequences corresponding to different isoforms within the same species, while highly conserved in sequences corresponding to the same isoform in different species. We found a number of such residues, and they are largely located at the antiparallel interfaces observed in the 3D structures. Our sequence analysis thus provides strong evidence that the observed interfaces indeed mediate the homophilic interactions important in self-avoidance.<br />
	<br />
	In a second complementary bioinformatics approach, we used protein sequence co-evolution to identify possible oligomeric contacts in clustered protocadherins. Residues in close proximity across important protein-protein interfaces co-evolve such that mutations on one side of the interface often result in compensatory mutations on the other. Such co-evolutionary analysis requires many sequences to get reliable signals. Here we took advantage of the fact that each vertebrate genome contains about 50 clustered protocadherin isoforms, allowing us to quantify these evolutionary correlations in a large multiple sequence alignment. Clustered protocadherins actually form one of the few protein families present only the animal kingdom for which co-evolutionary analysis is possible with the limited number of animal genomes available. Our analysis identified evolutionary correlations between repeats 2 and 3 and between repeats 1 and 4. The data for repeats 2 and 3 corroborate the interfaces we observed in our protein structures. Importantly, the data for repeats 1 and 4 provide further evidence of the antiparallel arrangement, and predict an interface consistent with an extension of our structures, which only contained repeats 1-3.<br />
	<br />
	In summary, our newly published study identifies the molecular basis for homophilic specificity in clustered protocadherin interactions that lead to neuronal self-avoidance. Future work will address the how several of these binary homophilic interactions are assembled at the cellular level to generate the full self-identity code in a developing neuron.</p>
<p>
	Read more in <a href="http://www.sciencedirect.com/science/article/pii/S0969212615003755">Structure</a> or download <a href="news_images/editor_uploads/2015/10/1-s2.0-S0969212615003755-main.pdf">PDF</a></p>
<p>
	&nbsp;</p>
<p>
	&nbsp;</p>
<p>
	&nbsp;</p>
<p>
	&nbsp;</p>

        </div><!-- end: story content -->
    </div>
    <!-- end: title row-->


</div><!-- end: main container -->
<div class="container" id="footer-container">
    <div class="row" id="news-footer">
        <div class="col-12 text-left">
                Copyright &copy; The President and Fellows of Harvard College. Created September 2018.
        </div>
    </div>
</div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->

    <!--script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script-->
    <script src="assets/js/vendor/jquery-slim.min.js"></script>
    <script src="assets/js/vendor/popper.min.js"></script>
    <script src="assets/bootstrap-4.1.3/dist/js/bootstrap.min.js"></script>
    <script src="assets/js/vendor/holder.min.js"></script>
    

  </body>
</html>
