<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="assets/favicon.ico">

    <title>Stamp Collecting Gets Important [Sanes Lab] - Molecular &amp; Cellular Biology - News Archives - Harvard University</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/bootstrap-4.1.3/dist/css/bootstrap.min.css" rel="stylesheet">

    <link href="assets/css/mcb_archive.css" rel="stylesheet">
    <style>
        .container {
            font-size:95%;
        }
    </style>
  </head>

  <body>
<div class="container">
    <div class="row" style="background-color:#c5cdd7; padding:10px 0;">
          <div class="col-12 text-center">
            <a class="blog-header-logo text-dark"  href="index.html">
                <img src="assets/images/mcb_header2.png" class="img-fluid" alt="Molecular and Cellular Biology" /></a>
          </div>
    </div>
</div>

<div class="container" id="news-page-body">

    <div class="row"  id="letter_or_year_menu">
    <div class="col-5 text-left">
        
            <a href="story-2016-3906-mcb-graduate-student-balances-lab-work-and-outreach.html">&lt;&lt;</a>
        
        &nbsp;
        
            <a href="story-2016-3906-mcb-graduate-student-balances-lab-work-and-outreach.html">MCB Graduate Student Balances Lab Work and Outreach</a>
        

    </div>
    <div class="col-2 text-center">
        &nbsp; | &nbsp;
    </div>
    <div class="col-5 text-right">
        
            <a href="story-2016-3907-panning-for-precious-metals-gaudet-lab.html">Panning for Precious Metals [Gaudet Lab]</a>
        
        &nbsp;
        
            <a href="story-2016-3907-panning-for-precious-metals-gaudet-lab.html">&gt;&gt;</a>
        
    </div>
</div>


    <div class="row">
        <div class="col-12 text-left" >
            <br />
            <p>
                <a href="news_year_2017.html">News Archives</a> :
                <a href="news_year_2016.html">2016</a> :
                Stamp Collecting Gets Important [Sanes Lab]
        </div>
    </div>


    <!-- title row -->
    <div class="row">
        <div class="col-12 text-left" >
            <h2 class="title">Stamp Collecting Gets Important [Sanes Lab]</h2>

            
            <p>by 
    
    Joshua Sanes
                
            </p>
            
            <p>August 25, 2016</p>
        </div>

        <!-- start: story content --><div class="col-12 text-left">
            <div class="FloatL" style="width:420px;">
	<img alt="" src="news_images/editor_uploads/2016/08/2016_sanes_cell_6967.png" style="width: 420px; height: 300px;" />Irene Whitney and <a href="https://www.mcb.harvard.edu/mcb/faculty/profile/joshua-r-sanes/">Josh Sanes</a></div>
<p>
	The complex circuits of the brain are built from thousands of types of neurons. More than a century ago, Santiago Ramon y Cajal founded what we now call &ldquo;neurobiology&rdquo; by using the best anatomical methods of the day to classify hundreds of these types, and map the connections among them.&nbsp; Subsequently, the classification enterprise lost its allure, coming to be seen as mere &ldquo;stamp collecting.&rdquo;&nbsp; Over the past decade, the tide has turned. There are new ways to identify molecular signatures that distinguish neurons from each other. Once neurons are characterized, new tools can be applied to mark and manipulate them.&nbsp; Moreover, new methods for relating neural circuitry to behavior rely on having an accurate parts list.&nbsp; And there is growing realization that many brain diseases will be understood and cured only when we know exactly which neuronal types they affect.&nbsp; For all these reasons, neuronal classification is a main focus of the ongoing BRAIN Initiative, a coordinated nation-wide effort to develop the new technologies needed to understand the human brain. On the other hand, technical obstacles remain, as well as conceptual ones: some troglodytes continue to question whether neurons can even be divided into distinct types.<br />
	<br />
	In a new <a href="http://www.sciencedirect.com/science/article/pii/S0092867416310078">paper</a> (<a href="news_images/editor_uploads/2016/08/1-s2.0-S0092867416310078-main.pdf">PDF</a>), we address these challenges by attempting to completely categorize a heterogeneous class of neurons called &ldquo;retinal bipolar cells.&rdquo;&nbsp; We used a method called Drop-seq that enables molecular (transcriptomic) profiling of single cells in huge numbers but at very low cost per cell.&nbsp; In an initial paper, Evan Macosko and colleagues (also authors of the new paper) developed the method and we helped them apply it to retina (<a href="http://www.sciencedirect.com/science/article/pii/S0092867415005498">Macosko et al</a>., Cell, 2015).&nbsp; Here, we asked whether it could be used for complete classification, and whether there was precise correspondence between molecular and morphological criteria. We profiled some 25,000 bipolar cells and used bioinformatic tools to divide them into discrete groups.&nbsp; We identified a total of 15 bipolar types, including all previously known types plus new ones.&nbsp;&nbsp; Each type defined molecularly has a distinct morphology, with no evidence for intermediate types or a continuum of identities. Interestingly, one of the new types had a very atypical morphology for bipolar cells: it is unipolar! It may have gone undetected or been misclassified in the past for this reason &ndash; yet we showed that it actually begins life as a typical bipolar, and retains its molecular characteristics as it undergoes a bizarre developmental shape change. This discovery illustrates the power of high throughput molecular classification, and raises the possibility that other similar &ldquo;hidden&rdquo; types may exist.<br />
	<br />
	Although a main purpose of this work was to pave the way for developmental and functional studies of visual circuitry, we also used bipolar cells to develop and validate a broadly useful pipeline for cell type classification.&nbsp; We systematically tested a variety of bioinformatics tools to find which were best at classifying cells, avoiding generation of spurious clusters or merging of distinct types. We asked how many cells need to be profiled for complete coverage, compared Drop-seq to other single cell profiling methods, provided new molecular markers for each type, and developed a method for determining the morphology of cells that express the marker genes. This suite of methods can now be applied to other cell classes throughout the brain.&nbsp; To get started, are are already using them to classify the other classes of retinal neurons into types, and beginning to ask whether specific types are selectively affected in conditions that lead to blindness such as nerve injury or glaucoma. &nbsp;<br />
	&nbsp;</p>
<p>
	Read more in <a href="http://www.sciencedirect.com/science/article/pii/S0092867416310078">Cell</a> or download <a href="news_images/editor_uploads/2016/08/1-s2.0-S0092867416310078-main_.pdf">PDF</a></p>

        </div><!-- end: story content -->
    </div>
    <!-- end: title row-->


</div><!-- end: main container -->
<div class="container" id="footer-container">
    <div class="row" id="news-footer">
        <div class="col-12 text-left">
                Copyright &copy; The President and Fellows of Harvard College. Created September 2018.
        </div>
    </div>
</div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->

    <!--script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script-->
    <script src="assets/js/vendor/jquery-slim.min.js"></script>
    <script src="assets/js/vendor/popper.min.js"></script>
    <script src="assets/bootstrap-4.1.3/dist/js/bootstrap.min.js"></script>
    <script src="assets/js/vendor/holder.min.js"></script>
    

  </body>
</html>
