<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="assets/favicon.ico">

    <title>MCB Welcomes Sean Eddy and Elena Rivas - Molecular &amp; Cellular Biology - News Archives - Harvard University</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/bootstrap-4.1.3/dist/css/bootstrap.min.css" rel="stylesheet">

    <link href="assets/css/mcb_archive.css" rel="stylesheet">
    <style>
        .container {
            font-size:95%;
        }
    </style>
  </head>

  <body>
<div class="container">
    <div class="row" style="background-color:#c5cdd7; padding:10px 0;">
          <div class="col-12 text-center">
            <a class="blog-header-logo text-dark"  href="index.html">
                <img src="assets/images/mcb_header2.png" class="img-fluid" alt="Molecular and Cellular Biology" /></a>
          </div>
    </div>
</div>

<div class="container" id="news-page-body">

    <div class="row"  id="letter_or_year_menu">
    <div class="col-5 text-left">
        
            <a href="story-2015-3826-josh-sanes-featured-in-science-magazine.html">&lt;&lt;</a>
        
        &nbsp;
        
            <a href="story-2015-3826-josh-sanes-featured-in-science-magazine.html">Josh Sanes Featured in Science Magazine</a>
        

    </div>
    <div class="col-2 text-center">
        &nbsp; | &nbsp;
    </div>
    <div class="col-5 text-right">
        
            <a href="story-2015-3828-hooking-up-to-detect-local-motion-sanes-lab.html">Hooking Up to Detect Local Motion [Sanes Lab]</a>
        
        &nbsp;
        
            <a href="story-2015-3828-hooking-up-to-detect-local-motion-sanes-lab.html">&gt;&gt;</a>
        
    </div>
</div>


    <div class="row">
        <div class="col-12 text-left" >
            <br />
            <p>
                <a href="news_year_2017.html">News Archives</a> :
                <a href="news_year_2015.html">2015</a> :
                MCB Welcomes Sean Eddy and Elena Rivas
        </div>
    </div>


    <!-- title row -->
    <div class="row">
        <div class="col-12 text-left" >
            <h2 class="title">MCB Welcomes Sean Eddy and Elena Rivas</h2>

            
            <p>by 
    
    Cassie Martin
                
            </p>
            
            <p>August 24, 2015</p>
        </div>

        <!-- start: story content --><div class="col-12 text-left">
            <div class="FloatL" style="width:400px;">
	<img alt="" src="news_images/editor_uploads/2015/08/_Eddy-&amp;-Rivas_2015.png" style="width: 400px; height: 320px;" /></div>
<p>
	<a href="http://eddylab.org/">Sean Eddy</a> and Elena Rivas are computational biologists at the top of their field. Hailing from the <a href="https://www.hhmi.org/">HHMI</a> funded <a href="https://www.janelia.org/">Janelia Research Campus</a> just outside of Washington D.C., the husband and wife duo will join the Department of Molecular and Cellular Biology this fall&mdash;Eddy as a professor of molecular and cellular biology, and Rivas as a senior research fellow and lecturer.</p>
<p>
	Over the past few decades, the scientific spotlight has shifted from DNA to RNA as researchers realized the molecule&rsquo;s increasing importance in evolution. Beyond biology&rsquo;s central dogma lies an entire world of undiscovered classes of RNA&mdash;molecular clues that could tell us more about evolutionary history and bring the tree of life into sharper focus. Known for their cutting-edge technology, Eddy and Rivas build and harness computational tools that analyze the structure, function, and evolutionary history of RNA sequences in modern genomes. Their goal: to understand how complex biological functions evolve from low-level changes in the genetic code.</p>
<p>
	&nbsp;</p>
<div class="FloatL" style="width:150px;">
	<img alt="" src="news_images/editor_uploads/2015/08/eddy_s_2015.png" style="width: 150px; height: 160px;" /></div>
<p>
	<strong>Sean Eddy</strong></p>
<p>
	Eddy&rsquo;s love of science was bolstered by a childhood spent in rural Pennsylvania exploring the outdoors and raising frogs and spiders&mdash;activities encouraged by his mom. &ldquo;My mom always wanted to be a biologist, which pushed me in the direction of science,&rdquo; Eddy said. But Mrs. Eddy put her dreams on hold to raise six kids. &ldquo;She joked she was going to live through me and the others,&rdquo; he said. When they were old enough, she went back to school. &ldquo;She was in graduate school at the same time as me, which means I got no sympathy whatsoever,&rdquo; Eddy joked. But her dedication&mdash;commuting hours from home to the lab, doing research while still taking care of her family&mdash;really inspired him. &ldquo;Nowadays when I hear grad students complain, I say &lsquo;My mother&hellip;!&rsquo; like going barefoot in the snow or something.&rdquo;</p>
<p>
	After high school Eddy attended the California Institute of Technology intending to study physics, but found the math too difficult. &ldquo;They had a program for recruiting disadvantaged kids and bringing them up to speed in physics and math,&rdquo; said Eddy. &ldquo;That was partially successful with me. The only class that I could at all keep up with was probability theory, which is sort of prophetic because that&#39;s what I ended up doing,&rdquo; he said. &ldquo;I found it very intuitive.&rdquo; Eddy eventually switched his major to biology. After graduating with a bachelor&rsquo;s degree in 1986, he took his talents to the University of Colorado at Boulder where he pursued a PhD in molecular biology.</p>
<p>
	At the time, a chemist at the university named Thomas Cech had just discovered self-splicing RNA through group I introns&mdash;large ribozymes that catalyze their own removal from RNA precursors in a wide range of organisms. Intrigued by the molecule and surrounded by groundbreaking RNA researchers, Eddy studied the movement of these parasitic introns in T4 bacteriophage. &ldquo;It was super weird because bacteria, and especially phage, aren&rsquo;t supposed to have introns,&rdquo; he said. Eddy&rsquo;s advisor Larry Gold theorized that the introns were really regulatory elements, but the genomes of phage such as T2 and T6 didn&rsquo;t contain these introns. If they were really regulatory elements, then the introns should be conserved across organisms. Perplexed, Eddy was determined to find more. &ldquo;I had gotten pretty good at folding the introns by eye from looking at the sequence, but very little sequence was conserved,&rdquo; he said. &ldquo;So there was an interesting computational problem, which was how do I go looking in a genome sequence for something that can fold in a pattern for an RNA as opposed to looking for it by sequence?&rdquo; After brainstorming with a few computational biologists over friendly games of basketball, Eddy began writing an algorithm. After toiling away for six months with no answer, however, he decided to move on&mdash;not just from the University of Colorado, but also from RNA.</p>
<p>
	Across the pond in Cambridge, UK, Eddy joined the lab of John Sulston at the MRC Laboratory of Molecular Biology in 1992 as a postdoctoral fellow. There he began working on how growing neurons in Caenorhabditis elegans find their targets. &ldquo;The danger of neuroscience is you gravitate toward these huge unsolvable problems. I was trying to figure out what would be an approachable problem in a model system,&rdquo; he said. &ldquo;I was attracted to the worm, to the people working on it, and to the way they thought about problems.&rdquo; Specifically, Eddy wanted to screen for animals that had mutations in the axon outgrowth pathways of the motor nervous system. Because the anatomy of C. elegans was so thoroughly documented, it was easy to hypothesize which neurons were defective just by observing how the worms moved. Eddy just needed to find a way to visualize the axons in living worms through a genetic screen. &ldquo;It seemed to me a great thing if you could use a fluorescent reporter that you could put into a particular class of neurons and then see those processes and select for worms who are wired wrong under a light microscope,&rdquo; he said. For months, Eddy tried using luciferase&mdash;a firefly protein that produces light when combined with luciferin and ATP&mdash;but that was a dead end. He eventually obtained green fluorescent protein (GFP), a new jellyfish protein that had the scientific community buzzing, but another scientist beat him to the punch. Martin Chalfie, a professor of biological sciences at Columbia University, published a study in Science where he conducted the same research Eddy planned to do using GFP. Chalfie would go on to win the Nobel Prize in 2008 for the discovery and development of the protein.</p>
<p>
	Not to be discouraged, Eddy turned his attention to a side project. In collaboration with his postdoc mentor Richard Durbin, Eddy created algorithms that would form the basis for his protein homology search software HMMER. But the RNA problem from his graduate years still plagued him. The pre-existing algorithms Eddy worked with had a hard time handling RNA because the molecule forms secondary structures that depend on base pairing rules. Over evolutionary time, when a nucleotide changes in a sequence involved in forming a secondary structure, the base it pairs with also changes. No algorithms existed that could deal with RNA&rsquo;s dynamic base-pairing relationships. After poring over the research again, &ldquo;there was a day I was sitting doodling in my notebook and I realized I had come up with the solution,&rdquo; he said. &ldquo;It&rsquo;s one of those things that suddenly snaps together for you.&rdquo; Excited, he triple checked his work and showed it to one of his co-advisors, who informed him that his algorithm wasn&rsquo;t new at all&mdash;it was a stochastic context free grammar, a formalism that&rsquo;s been used in computational linguistics for years. Fortunately for Eddy, the algorithm was relatively new to the world of RNA research&mdash;Yasu Sakakibara of Keio University in Tokyo, Japan, had a very similar idea around the same time, and the two men eventually published coordinated papers on the topic. Now Eddy had a decision to make. &ldquo;[This project] was really new and it was hard to imagine a bunch of people jumping onto this RNA problem, which had always fascinated me. It was really easy to say I&rsquo;m just going to put all my efforts into probability models for sequence analysis,&rdquo; he said.</p>
<p>
	By the time Eddy joined the faculty at Washington University in 1995, he was fully committed to sequence analysis. His lab was involved in analyzing the C. elegans genome sequence as well as annotating the mouse and human genomes. Eddy continued developing HMMER and the RNA algorithm&mdash;now called Infernal&mdash;which allows users to search genome sequences for RNAs that have conserved secondary structures. Now that he had the tools, Eddy began searching for the catalytic RNA that fascinated him in graduate school. Instead, he discovered dozens of small nucleolar RNAs (snoRNAs)&mdash;a class of RNA molecules that guide chemical modifications of ribosomal and transfer RNA&mdash;in yeast and archaeal genomes. During this time, Eddy also met Elena Rivas, his colleague and future wife.</p>
<p>
	&nbsp;</p>
<div class="FloatL" style="width:150px;">
	<img alt="" src="news_images/editor_uploads/2015/08/rivas_e_2015.png" style="width: 150px; height: 160px;" /></div>
<p>
	<strong>Elena Rivas</strong></p>
<p>
	For years, Rivas had been a particle physicist both in her native Spain and in the U.S. working on lattice field theory&mdash;a field of quantum physics that assumes time and space are a network of discrete points. But she eventually became disillusioned. &ldquo;Physics is really hard and there is not much immediate reward for it because it&#39;s very difficult to test anything, especially if you work at really high energies,&rdquo; she said. Rivas is a problem solver, and enjoys following a question through to its conclusion. Theoretical physics just didn&rsquo;t fulfill that need.</p>
<p>
	Unsure of what to do next, Rivas thought that medical school might be a good fit as she had an interest in the subject as a young student. But before she committed to a drastic career change, some friends encouraged her to work in the biological field, which is how she landed in Eddy&rsquo;s lab. As a Sloan fellow, Rivas had her own funding and worked alongside Eddy. Her first project in the lab bridged her physics skills and computational biology. Rivas was studying a complicated RNA structure called a pseudoknot that Eddy&rsquo;s algorithms couldn&rsquo;t handle. While Eddy was struggling with the code, Rivas immediately made a connection to Feynman diagrams&mdash;a way to calculate cross sections for colliding particles&mdash;which resolved the issue. &ldquo;It&rsquo;s kind of like a graphical way of remembering all the steps,&rdquo; she said.</p>
<p>
	From then on Rivas was hooked; she left medical school behind for good and continued working in Eddy&rsquo;s lab. &ldquo;A major component of my work has been working with RNA genes, which don&rsquo;t produce proteins but work as RNAs,&rdquo; she said. &ldquo;The RNA sequence folds into little helices and has a defined structure, which is really important for the function of the RNA. I try to elucidate the structure from data.&rdquo; Rivas developed one of the first RNA gene finding programs, which she used to predict genes in E.coli, archaea, yeast, and the nematode worm. That research sparked her interest in evolution; it wasn&rsquo;t enough to just find these genes, she had to know if they were important.</p>
<p>
	By then, Eddy and Rivas had moved to Janelia. The dedicated systems neuroscience research facility is fully funded by HHMI and is home to a small group of scientists. Eddy and Rivas jumped at the chance to join Janelia, even though systems neuroscience wasn&rsquo;t exactly their focus. Unlike large universities, Janelia not only provides full funding for researchers, but the space and time for them to do science with minimal distractions.</p>
<p>
	At Janelia, Rivas continued developing experimental codes and software packages for interpreting genome sequences. Based on probabilistic models first developed by linguists to quantify language, Rivas&rsquo; models not only identified structures, they also compared sequences across organisms, ferreted out hidden conserved genes by searching for insertion and deletions in the sequences, and amplified weak signals of noncoding RNA&mdash;a functional RNA molecule that isn&rsquo;t translated into protein, but that many people think may be evolutionarily important. &ldquo;Anything we can think about evolution is going to help us relate genes from different organisms, which could help us understand mechanisms,&rdquo; Rivas said. &ldquo;A mechanism could be very complicated in humans, but if you manage to find a homolog of that gene, say in bacteria, it&#39;s going to help you make a more complicated process simpler so you can understand it.&rdquo;</p>
<p>
	While Rivas worked on RNA, Eddy managed a handful of projects. He continued accelerating the performance of Infernal and other programs, and became involved in cell-type specific genomics. With the help of postdocs Lee Henry and Fred Davis, Eddy&rsquo;s lab developed a technique for purifying nuclei from a targeted group of cells using epitope tags. They are using the technology to study the optic lobe of the fruit fly Drosophila. Using GAL4 &ldquo;driver lines&rdquo; that label individual cell types, developed by Aljoscha Nern and Gerry Rubin, the team is sequencing the transcriptome of each cell-type in the circuitry of the fly eye. The information should help other groups at Janelia map and understand that circuitry. But perhaps Eddy&rsquo;s most important accomplishment was developing his HMMER software into a web server that is competitive with BLAST&mdash;the go-to online homology search engine. Hosted by the European Bioinformatics Institute in Cambridge, UK, the search engine went live in June.</p>
<p>
	The launch marks the end of Eddy and Rivas&rsquo; time at Janelia. &ldquo;I feel like we made a really good run on particular problems that I needed to solve, but I&#39;m coming to the end of that and felt like I needed new ideas,&rdquo; said Eddy. The scientists are bound to find plenty of them at MCB. &ldquo;The department is very diverse, with many people doing cool stuff,&rdquo; Rivas said. Eddy and Rivas are currently building a lab and looking forward to developing new projects and collaborations. With no set research plans yet, they will continue refining their current software and building tools to answer new questions as they arise. &ldquo;I&#39;m open to looking at other problems, to working on interesting problems whether it&rsquo;s RNA or something else,&rdquo; she said.</p>
<p>
	Although they&rsquo;re still getting settled, Eddy is no stranger to MCB. He served on the advisory panel for the FAS Systems Biology Center for Andrew Murray and Erin O&rsquo;Shea, and watched from the sidelines as the Bauer Fellows Program developed. &ldquo;I draw the comparison very often to Janelia,&rdquo; Eddy said. &ldquo;It was just lovely, and each fellow was working on something near and dear to my heart.&rdquo; Eddy also made friends with Sharad Ramanathan. They tried to find ways to work together over the years, but it never worked out. Now they finally have their chance. &ldquo;Because of that history, MCB has always felt like a second home to me,&rdquo; Eddy said. To that we say, welcome home.</p>

        </div><!-- end: story content -->
    </div>
    <!-- end: title row-->


</div><!-- end: main container -->
<div class="container" id="footer-container">
    <div class="row" id="news-footer">
        <div class="col-12 text-left">
                Copyright &copy; The President and Fellows of Harvard College. Created September 2018.
        </div>
    </div>
</div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->

    <!--script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script-->
    <script src="assets/js/vendor/jquery-slim.min.js"></script>
    <script src="assets/js/vendor/popper.min.js"></script>
    <script src="assets/bootstrap-4.1.3/dist/js/bootstrap.min.js"></script>
    <script src="assets/js/vendor/holder.min.js"></script>
    

  </body>
</html>
