<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="assets/favicon.ico">

    <title>CRISPR Cell Lineages [Schier Lab] - Molecular &amp; Cellular Biology - News Archives - Harvard University</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/bootstrap-4.1.3/dist/css/bootstrap.min.css" rel="stylesheet">

    <link href="assets/css/mcb_archive.css" rel="stylesheet">
    <style>
        .container {
            font-size:95%;
        }
    </style>
  </head>

  <body>
<div class="container">
    <div class="row" style="background-color:#c5cdd7; padding:10px 0;">
          <div class="col-12 text-center">
            <a class="blog-header-logo text-dark"  href="index.html">
                <img src="assets/images/mcb_header2.png" class="img-fluid" alt="Molecular and Cellular Biology" /></a>
          </div>
    </div>
</div>

<div class="container" id="news-page-body">

    <div class="row"  id="letter_or_year_menu">
    <div class="col-5 text-left">
        
            <a href="story-2016-3891-annual-boston-bacterial-meeting-to-be-held-at-mit-june-14-15.html">&lt;&lt;</a>
        
        &nbsp;
        
            <a href="story-2016-3891-annual-boston-bacterial-meeting-to-be-held-at-mit-june-14-15.html">Annual Boston Bacterial Meeting to be Held at MIT June 14-15</a>
        

    </div>
    <div class="col-2 text-center">
        &nbsp; | &nbsp;
    </div>
    <div class="col-5 text-right">
        
            <a href="story-2016-3893-turning-up-the-heat-on-protein-folding-by-turning-down-heat-shock-factor-denic-lab.html">Turning Up the Heat on Protein Folding by Turning Down Heat Shock Factor [Denic Lab]</a>
        
        &nbsp;
        
            <a href="story-2016-3893-turning-up-the-heat-on-protein-folding-by-turning-down-heat-shock-factor-denic-lab.html">&gt;&gt;</a>
        
    </div>
</div>


    <div class="row">
        <div class="col-12 text-left" >
            <br />
            <p>
                <a href="news_year_2017.html">News Archives</a> :
                <a href="news_year_2016.html">2016</a> :
                CRISPR Cell Lineages [Schier Lab]
        </div>
    </div>


    <!-- title row -->
    <div class="row">
        <div class="col-12 text-left" >
            <h2 class="title">CRISPR Cell Lineages [Schier Lab]</h2>

            
            <p>by 
    
    Alexander Schier
                
            </p>
            
            <p>June 13, 2016</p>
        </div>

        <!-- start: story content --><div class="col-12 text-left">
            <p>
	The cells in a multicellular organism develop via a lineage that starts with the fertilized egg. Lineage trees describe the divisions and specializations of cells over time, as exemplified in the classic <em>C. elegans</em> lineage tree constructed in the late 1970s that depicts each of the ~1000 cell divisions leading from egg to adult. For more complex organisms with millions or billions of cells, it has been impossible to construct large-scale lineage trees. Now the <a href="http://www.schierlab.fas.harvard.edu/">Schier</a> and <a href="http://krishna.gs.washington.edu/">Shendure</a> labs (University of Washington) report in <a href="http://science.sciencemag.org/content/early/2016/05/25/science.aaf7907">Science</a> a method that uses the cumulative and combinatorial editing of a short genomic barcode to re-construct lineage trees of unprecedented scale and complexity.</p>
<div class="FloatL" style="width:360px;">
	<img alt="" src="news_images/editor_uploads/2016/06/jg+as_6401.png" style="width: 360px; height: 300px;" />James Gagnon (l) and Alex Schier</div>
<p>
	The approach, called GESTALT for genome editing of synthetic target arrays for lineage tracing, rests on three foundations. First, diverse DNA barcodes can be used to uniquely mark cells and can be easily read out by sequencing. Second, mutations introduced in the DNA of a cell are inherited by its descendants, allowing the re-construction of lineage relationships. Third, the CRISPR/Cas9 system can be used to introduce mutations by targeting guide RNAs to specific sites in the genome and introducing DNA double-strand breaks. James Gagnon in the Schier lab with Aaron McKenna and Greg Findlay in the Shendure lab combined these three principles to introduce a barcode consisting of ten guide RNA binding sites into the genome and then used CRISPR/Cas9 genome editing to generate and accumulate diverse barcode mutations over multiple cell divisions. Over time, thousands of different barcode mutations were generated that were then read out from individual cells by sequencing. The mutated barcode sequences could then be compared to each other to reconstruct the lineage relationships between cells. For example, a cell with barcode mutations A+B+C is more closely related to a cell with mutations A+B+E than to a cell with mutations A+F+G.</p>
<p>
	The authors applied GESTALT to zebrafish development. They introduced Cas9 and ten guide RNAs into a barcode-containing embryo at the one-cell stage. Over a few hours, thousands of barcode edits accumulated during early embryogenesis. Growing these embryos to adulthood resulted in fish with thousands of differently mutated barcodes. Sequencing these barcodes from more than 200,000 cells and different organs revealed that the barcodes could be used to reveal the lineage relationships between cells and that, as expected from classical embryological studies, progenitor cells initially give rise to most tissue types but as development progresses their descendants become restricted to specific germ layers and organs. In addition, the study revealed that very few embryonic progenitors give rise to the majority of the cells in different tissues. For example, in one fish 5 progenitors gave rise to more than 98% of blood cells and 40 different progenitors gave rise to more than 90% of cells in the adult brain. These observations lay the foundation for future studies to determine how some progenitors come to dominate tissue contribution, how cells become more and more restricted in their fate as development proceeds, and how adult cells maintain and regenerate different organs. The same approach can also be used to study the lineage relationships of cells in tumors and metastases or of immune cells in the course of an infection.<br />
	<br />
	Future applications of GESTALT include the generation of complete maps of cell lineage in any organism whose genome can be edited by CRISPR/Cas9. This technology could also be adapted to connect lineage relationships to transcriptional or chromatin profiles. Other applications include the coupling of editing activity to signaling pathways to record the exposure of cells to different signals over time in the edits introduced in the barcode. Thus, the cumulative and combinatorial of editing of barcodes can be used to stably record many classes of biological information.</p>
<div class="FloatL" style="width:360px;">
	<img alt="" src="news_images/editor_uploads/2016/06/as_st_6632.png" style="width: 360px; height: 300px;" />Alex Schier (l) and Summer Thyme</div>
<p>
	Two additional papers in the CRISPR/Cas9 field were recently published by Summer Thyme in the Schier lab. The study in <a href="http://www.cell.com/cell-reports/fulltext/S2211-1247%2816%2930553-8">Cell Reports</a> shows that the double-strand breaks introduced by CRISPR/Cas9 and other mutagenic agents in zebrafish embryos are not repaired by classical non-homologous end joining, as widely assumed. Instead, double-strand breaks are repaired by an alternative non-homologous end-joining pathway that uses DNA microhomologies. A similar mechanism might also occur in mammalian embryos, because the sequence features of regions repaired after CRISPR/Cas9 cleavage resemble those found in zebrafish embryos. The study in Nature Communications analyzes why some guide RNAs induce no or little cleavage in zebrafish embryos. Thyme et al. find two reasons: some short stretches in the genome are resistant to CRISPR/Cas9 cleavage, and intramolecular interactions within guide RNAs can interfere with the ability of guide RNA-Cas9 complexes to generate double-strand breaks in DNA. These findings will improve the design of high-activity guide RNAs.</p>
<p>
	Read more in:</p>
<p>
	<a href="http://science.sciencemag.org/content/early/2016/05/25/science.aaf7907.full">Science</a>, <a href="news_images/editor_uploads/2016/06/science.aaf7907.full.pdf">PDF</a></p>
<p>
	<a href="http://www.cell.com/cell-reports/fulltext/S2211-1247%2816%2930553-8">Cell Reports</a>, <a href="news_images/editor_uploads/2016/06/PIIS2211124716303618.pdf">PDF</a>, corrected <a href="news_images/editor_uploads/2016/06/PIIS2211124716305538-1.pdf">PDF</a></p>
<p>
	<a href="http://www.nature.com/ncomms/2016/160610/ncomms11750/full/ncomms11750.html">Nature Communications</a>, <a href="news_images/editor_uploads/2016/06/ncomms11750.pdf">PDF</a></p>
<p>
	<a href="http://www.sciencemag.org/news/2016/05/genome-editor-crispr-helps-trace-growth-embryos-and-maybe-cancer-next">Science Magazine</a></p>
<p>
	<a href="http://www.bbc.com/news/science-environment-36397034">BBC NEWS Science &amp; Environment</a></p>
<p>
	<a href="http://www.the-scientist.com/?articles.view/articleNo/46173/title/Editing-Genomes-to-Record-Cellular-Histories/">The Scientist</a></p>
<p>
	<a href="http://news.harvard.edu/gazette/story/2016/08/finding-biological-barcodes/?utm_source=SilverpopMailing&amp;utm_medium=email&amp;utm_campaign=08.24.2016%20%281%29">Harvard Gazette</a></p>
<p>
	&nbsp;</p>
<p>
	&nbsp;</p>
<p>
	&nbsp;</p>
<p>
	&nbsp;</p>

        </div><!-- end: story content -->
    </div>
    <!-- end: title row-->


</div><!-- end: main container -->
<div class="container" id="footer-container">
    <div class="row" id="news-footer">
        <div class="col-12 text-left">
                Copyright &copy; The President and Fellows of Harvard College. Created September 2018.
        </div>
    </div>
</div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->

    <!--script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script-->
    <script src="assets/js/vendor/jquery-slim.min.js"></script>
    <script src="assets/js/vendor/popper.min.js"></script>
    <script src="assets/bootstrap-4.1.3/dist/js/bootstrap.min.js"></script>
    <script src="assets/js/vendor/holder.min.js"></script>
    

  </body>
</html>
