<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="assets/favicon.ico">

    <title>New Structural RNAs? Now We Can Tell [Rivas and Eddy Labs] - Molecular &amp; Cellular Biology - News Archives - Harvard University</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/bootstrap-4.1.3/dist/css/bootstrap.min.css" rel="stylesheet">

    <link href="assets/css/mcb_archive.css" rel="stylesheet">
    <style>
        .container {
            font-size:95%;
        }
    </style>
  </head>

  <body>
<div class="container">
    <div class="row" style="background-color:#c5cdd7; padding:10px 0;">
          <div class="col-12 text-center">
            <a class="blog-header-logo text-dark"  href="index.html">
                <img src="assets/images/mcb_header2.png" class="img-fluid" alt="Molecular and Cellular Biology" /></a>
          </div>
    </div>
</div>

<div class="container" id="news-page-body">

    <div class="row"  id="letter_or_year_menu">
    <div class="col-5 text-left">
        
            <a href="story-2016-3932-insights-into-causes-of-anemia-from-a-membrane-protein-structure-gaudet-lab.html">&lt;&lt;</a>
        
        &nbsp;
        
            <a href="story-2016-3932-insights-into-causes-of-anemia-from-a-membrane-protein-structure-gaudet-lab.html">Insights into Causes of Anemia from a Membrane Protein Structure [Gaudet Lab]</a>
        

    </div>
    <div class="col-2 text-center">
        &nbsp; | &nbsp;
    </div>
    <div class="col-5 text-right">
        
            <a href="story-2016-3934-dan-needleman-awarded-tenure-faculty-position-in-applied-physics-and-molecular-and-cellular-biology.html">Dan Needleman Awarded Tenure Faculty Position in Applied Physics and Molecular and Cellular Biology</a>
        
        &nbsp;
        
            <a href="story-2016-3934-dan-needleman-awarded-tenure-faculty-position-in-applied-physics-and-molecular-and-cellular-biology.html">&gt;&gt;</a>
        
    </div>
</div>


    <div class="row">
        <div class="col-12 text-left" >
            <br />
            <p>
                <a href="news_year_2017.html">News Archives</a> :
                <a href="news_year_2016.html">2016</a> :
                New Structural RNAs? Now We Can Tell [Rivas and Eddy Labs]
        </div>
    </div>


    <!-- title row -->
    <div class="row">
        <div class="col-12 text-left" >
            <h2 class="title">New Structural RNAs? Now We Can Tell [Rivas and Eddy Labs]</h2>

            
            <p>by 
    
    Elena Rivas
                
            </p>
            
            <p>November 21, 2016</p>
        </div>

        <!-- start: story content --><div class="col-12 text-left">
            <div class="FloatL" style="width:400px;">
	<img alt="" src="news_images/editor_uploads/2016/11/2016_eddy-rivas._7797.png" style="width: 400px; height: 300px;" /><a href="https://www.mcb.harvard.edu/mcb/faculty/profile/elena-rivas/">Elena Rivas</a> (l) and <a href="https://www.mcb.harvard.edu/mcb/faculty/profile/sean-eddy/">Sean Eddy</a></div>
<p>
	Some fundamental cellular activities are directed by RNA-based machines. The RNAs in these machines adopt distinct three-dimensional structures in order to facilitate their function, much in the way that proteins fold into structures. For example, ribosomes, the large machines that produce our proteins, are mostly made of complex RNA structures. Other structural RNAs involved in fundamental cellular functions include the transfer RNAs, which recruit the building blocks that make proteins, and the spliceosomal RNAs, which are part of the machine that splices introns out of messenger RNAs. There are even examples of RNAs in large RNA-protein complexes for which we have little idea of their function, such as the Vault RNAs.</p>
<p>
	A signature of RNA structure that is clearly identifiable computationally is the co-variation of the nucleotide bases that come into contact (&ldquo;are base-paired&rdquo;) in the structure. Because pairs involving A:U and G:C (and to some extent G:U) are the most stable, two positions that are structurally paired tend to only allow mutations through evolution that preserve the RNA base pairing rules. Analyses of this kind were fundamental in computationally elucidating the base-paired structure of ribosomal RNAs to great precision before the three-dimensional structure was determined by crystallography. Thus, we expect that biologically relevant RNA structures would have the telltale signature of sequence covariation.</p>
<p>
	Over the last decade or so,&nbsp; now widely used RNA sequencing techniques have given us the ability to take a census of all RNAs (whether structural or not) present inside cells. These RNA-seq experiments have told us a consistent tale, that there are many more RNAs lurking inside a cell than one might have anticipated based on our best knowledge of the number of genes. They were called long noncoding RNAs (lncRNAs), to distinguish them from other numerous short regulatory RNAs in cells that are also being studied intensively, such as microRNAs. Why are there thousands of lncRNAs? How many of them have a function, as opposed to biochemical noise in the system?&nbsp; How many of them are structural, as opposed to functioning as linear RNAs, or being protein-coding mRNAs that we&rsquo;ve missed?</p>
<p>
	It is not an easy task to elucidate what even one lncRNA does. Perhaps the best studied lncRNA is Xist, which is responsible for the inactivation of one of the two X chromosomes in mammalian female cells. Xist has been known for more than 20 years, but still its mechanism of action is largely unknown. We do not even know if Xist functions through a conserved RNA structure, or through its conserved RNA linear sequence. If we could show that Xist does have a conserved RNA structure, knowledge of that structure would help us design more focused experiments to probe its molecular mechanism.</p>
<p>
	A lot of effort has been put into discerning possible base-paired structures in several lncRNAs. But even random sequences can be folded into plausible-looking structures, and RNA folding predictions are largely unable to distinguish structural RNAs from nonstructural RNAs, even with the most recent methods that use a combination of computational and experimental techniques. A key question is whether a proposed lncRNA structure shows the telltale sequence covariation evidence that the structure is important enough to be conserved through evolution. The computational tools used by lncRNA researchers have been incapable of telling when a covariation signal is strong enough to be evidence for an RNA structure, as opposed to background noise caused by fluctuations in sequence evolution.</p>
<p>
	In this work, Rivas and colleagues present a computational tool to test for an evolutionarily conserved RNA structure, by rigorously quantitating the statistical strength of observed pairwise covariations, while also being fast and easy for biologists to use on any RNA alignment they want to study. This new method takes into account one of the most confounding sources of false covariation signals, that the common origin of biological sequences produces covariations that could be confounded with covariations due to RNA structure.&nbsp; The method, <strong>R-scape</strong> (RNA Structural Covariation Above Phylogenetic Expectation) pinpoints the covariations that can truly be considered as statistical evidence for a conserved RNA structure.</p>
<p>
	Interestingly, for structures that had previously been proposed for three of the best studied lncRNAs -- Xist, HOTAIR, and steroid receptor activator RNA &ndash; reassessment of those structures with R-scape show that none of the covariation evidence that was used to support them is statistically significant above background. These results reopen the debate about whether these three lncRNAs have a function that depends on a conserved structure, and it will prompt reexamination of the evidence for conserved structure in other lncRNAs. R-scape contributes an important new computational tool for studying lncRNAs.</p>
<p>
	Read more in <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4066.html">Nature Methods</a>, <a href="news_images/editor_uploads/2016/11/nmeth.4066.pdf">PDF</a></p>
<p>
	&nbsp;</p>
<p>
	&nbsp;</p>
<p>
	&nbsp;</p>
<p>
	&nbsp;</p>
<p>
	&nbsp;</p>

        </div><!-- end: story content -->
    </div>
    <!-- end: title row-->


</div><!-- end: main container -->
<div class="container" id="footer-container">
    <div class="row" id="news-footer">
        <div class="col-12 text-left">
                Copyright &copy; The President and Fellows of Harvard College. Created September 2018.
        </div>
    </div>
</div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->

    <!--script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script-->
    <script src="assets/js/vendor/jquery-slim.min.js"></script>
    <script src="assets/js/vendor/popper.min.js"></script>
    <script src="assets/bootstrap-4.1.3/dist/js/bootstrap.min.js"></script>
    <script src="assets/js/vendor/holder.min.js"></script>
    

  </body>
</html>
